src/EventListener/PimcoreLoadOptimization.php

```diff
@@ -52,4 +54,9 @@ public function afterCommand(): void
         $this->versionEnabled && Version::enable();
         $this->cacheEnabled && Cache::enable();
     }
+
+    public function afterExecute(): void
+    {
+        \Pimcore::collectGarbage();
```

Comment:
Is it really necessary to do this after the execution of *every* fixture?
Calling this also has an overhead, right?

Reply:
Also: is this listener only called for the commands, or also when loading fixtures in tests? And if so: do we want this behavior for tests?

Reply:
Actually calling collectGarbage() does not seem to have much impact. Instead it is useful to disable debug mode and setting the env to prod when running the command. Calling collectGarbage() does not seem to be necessary or useful at this point.